### DISCUSSION TITLE
How do I remove the sharing of gradio in public url?

### DISCUSSION THREAD

**Discussion by supersuper1234** [InternLM/lmdeploy#515] [Q&A]
Hi, i am using the models locally and don't require a public url. may i knw which file and code should i edit to remove the sharing? thanks

**Comment by lvhan028** [comment]
@AllentDan any ideas?

**Comment by AllentDan** [comment]
Set all `share=True`  in [app.py](https://github.com/InternLM/lmdeploy/blob/b58a9dffb12bdc36ee9b3f251185179602468de9/lmdeploy/serve/gradio/app.py) to `share=False`
